src/cli.ts

```diff
@@ -176,7 +176,7 @@ function networkFromOptions(options: OptionValues): string {
   return network
 }
 
-function getOptions(program: Command, options: OptionValues): OptionValues {
+export function getOptions(program: Command, options: OptionValues): OptionValues {

```

Comment:
since the function is export please add function description in comment as documentation .